docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -678,9 +689,120 @@ This could be done by assigning unique (or practically unique, with very low pro
+* The "at risk" population for GQ person moves is people living in GQ.
+* The "at risk" population for non-reference-person moves is people living in non-GQ households who
+  are not the reference person in their household.
+
+In order to mitigate the sampling noise in ACS PUMS stratified by all of these demographic characteristics,
+we calculate the denominator for the rate from a resample of the "at risk" population, with perturbation.
+
+Simulation strategy
+^^^^^^^^^^^^^^^^^^^
+
+Emigration events are modeled as happening to an at-risk population at a certain rate.
+They are constant across time in the simulation.
+
+Households and individuals selected to have emigration events should remain in the simulation, but their
+state and PUMA attributes should be set to a placeholder value that signifies they are
+no longer in the US.
+Emigrating simulants should also have their address attribute removed, and they should terminate
+employment.
+In the future, we may want to sample some of these simulants for re-entry in the immigration component, but for now
+they will remain outside the US permanently.
+
+Household moves
+'''''''''''''''
+
+The at-risk population for household moves is all simulants living in non-GQ households in the US.
+This at-risk population should be stratified by age group, sex, race/ethnicity, and nativity (born in or outside the US)
+**of the simulant's household's reference person**, as well as US state.
+On each time step, within each stratum, the corresponding household move emigration rate **per year of person-time** should be applied to determine
+a number of **simulants** to emigrate as part of household moves.
+Then, households within the stratum should be sampled at random for emigration until the desired number of simulants is reached.
+
+GQ person moves
+'''''''''''''''
+
+The at-risk population for GQ person moves is all simulants living in GQ in the US.
+This at-risk population should be stratified by age group, sex, race/ethnicity, nativity (born in or outside the US),
+and US state.
+On each time step, within each stratum, the corresponding GQ person move emigration rate **per year of person-time**
+should be applied to sample simulants to emigrate. 
+
+Non-reference-person moves
+''''''''''''''''''''''''''
+
+The at-risk population for GQ person moves is all simulants living in non-GQ households in the US, except for those who are a household reference person.
+This at-risk population should be stratified by age group, sex, race/ethnicity, nativity (born in or outside the US),
+and US state.
+On each time step, within each stratum, the corresponding non-reference-person move emigration rate **per year of person-time**
+should be applied to sample simulants to emigrate.
+The simulant is removed from the household (they may be given a blank or placeholder household ID) and the
+rest of the household is unaffected by this event.
+
```

Comment:
Do we want to consider any kind of relationships in this? For example, it might be less likely for a spouse to emigrate alone than another person. But this is likely unneeded complication.

Reply:
Good point. We will get some of this by age stratification (age is likely correlated with relationship) but the specific example you mention is likely to happen too often.

As I've currently implemented this, we already have serious computational challenges calculating the rates, so adding stratification by relationship would be difficult.